reference/resources/globalnetworkpolicy.md

```diff
@@ -63,27 +63,27 @@ spec:
+| serviceAccountSelector | Selects the service account(s) to which this policy applies. Select all service accounts in the cluster with a specific name using the `projectcalico.org/name` label.                                                               |                     | [selector](#selectors) | all()                                         |
+| namespaceSelector      | Selects the namespace(s) to which this policy applies. Select a specific namespace by name using the `projectcalico.org/name` label.                                                                                                 |                     | [selector](#selectors) | all()                                         |
 | types                  | Applies the policy based on the direction of the traffic. To apply the policy to inbound traffic, set to `Ingress`. To apply the policy to outbound traffic, set to `Egress`. To apply the policy to both, set to `Ingress, Egress`. | `Ingress`, `Egress` | List of strings       | Depends on presence of ingress/egress rules\* |
 | ingress                | Ordered list of ingress rules applied by policy.                                                                                                                                                                                     |                     | List of [Rule](#rule) |                                               |
 | egress                 | Ordered list of egress rules applied by this policy.                                                                                                                                                                                 |                     | List of [Rule](#rule) |                                               |
 | doNotTrack\*\*         | Indicates to apply the rules in this policy before any data plane connection tracking, and that packets allowed by these rules should not be tracked.                                                                                | true, false         | boolean               | false                                         |
 | preDNAT\*\*            | Indicates to apply the rules in this policy before any DNAT.                                                                                                                                                                         | true, false         | boolean               | false                                         |
 | applyOnForward\*\*     | Indicates to apply the rules in this policy on forwarded traffic as well as to locally terminated traffic.                                                                                                                           | true, false         | boolean               | false                                         |
 
-\* If `types` has no value, {{site.prodname}} defaults as follows.
+If `types` has no value, {{site.prodname}} defaults as follows.
```

Comment:
The `\*` correlated to the table text above `Depends on presence of ingress/egress rules\*`.  So if you're removing the asterisk here, it would make sense to remove it in the table as well.

Does it need removing?  Maybe better to keep as it was.

Reply:
Ah, I couldn't figure out why it was there, though it was junk!